test/missing_semicolon.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title></title>
+    <script>
+        var xmlHttp = new XMLHttpRequest();
+        xmlHttp.open( "GET", '../src/js.cookie.js', false );
+        xmlHttp.send( null );
+        window.scriptContent = '(function (){ return {}; })() ';
+        window.scriptContent += xmlHttp.responseText;
+        window.scriptContent += ';window.ok = true;';
+
+        var script = document.createElement('script');
```

Comment:
Same as [this comment](https://github.com/js-cookie/js-cookie/pull/171/files#r59530280)